### ISSUE TITLE
error[E0658]: use of unstable library feature 'core_intrinsics'

### ISSUE THREAD

**Issue by jessebraham** [esp-rs/esp-println#8]
The newest `nightly` seems to have broken this:

```text
error[E0658]: use of unstable library feature 'core_intrinsics': intrinsics are unlikely to ever be stabilized, instead they should be used through stabilized interfaces in the rest of the standard library
  --> /Users/jesse/.cargo/registry/src/github.com-1ecc6299db9ec823/esp-println-0.2.0/src/rtt.rs:64:9
   |
64 |         core::intrinsics::copy_nonoverlapping(
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add `#![feature(core_intrinsics)]` to the crate attributes to enable

For more information about this error, try `rustc --explain E0658`.
error: could not compile `esp-println` due to previous error
warning: build failed, waiting for other jobs to finish...
```

**Comment by bjoernQ** [comment]
That should be fixed in 0.2.1

**Comment by jessebraham** [comment]
Ahh didn't realize that was out, had to `cargo update`. Thanks!
